portugues/amigo/tecnologia.md

```diff
@@ -0,0 +1,5 @@
+Você falou com seu amigo e...
+
+[O seu amigo pede para parar de incomodar toda vez que tem um problema no celular.](incomodar/aplicativo.md)
+
+[O seu amigo gostou muito da aplicação e decidiu lhe ensinar como colaborar com o desenvolvedor.](../colabora/colaborar.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.